cmake/modules/SwiftSupport.cmake

```diff
@@ -61,6 +61,30 @@ function(get_swift_host_os result_var_name)
   endif()
 endfunction()
 
+# Attempt to get the module triple from the Swift compiler.
+set(module_triple_command "${CMAKE_Swift_COMPILER}" -print-target-info)
+if(CMAKE_Swift_COMPILER_TARGET)
+  list(APPEND module_triple_command -target ${CMAKE_Swift_COMPILER_TARGET})
+endif()
+execute_process(COMMAND ${module_triple_command}
+  OUTPUT_VARIABLE target_info_json)
+string(JSON module_triple GET "${target_info_json}" "target" "moduleTriple")
+
+# If we failed to get the module triple, fall back to the host arch.
+if(NOT module_triple)
+  get_swift_host_arch(host_arch)
+  set(module_triple ${host_arch})
+  message(WARNING 
```

Comment:
Perhaps `SEND_ERROR` here? I think I'd prefer to fail the build if we can't get the appropriate filename.
If someone really wants to set the triple by hand, with the `if(NOT Swift_MODULE_TRIPLE)` check above, they can pass one on the command line easily enough.

Reply:
I used `FATAL_ERROR` here. Let me know if you'd prefer `SEND_ERROR`. I don't think there's any point in continuing here otherwise.